### PR TITLE
Update README.md Typo

### PR DESCRIPTION
**Pull request by kamilkrukowski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 ## Key Features
 
-* Backtesing exchange so you can try your trading strategies before using real funds.
+* Backtesting exchanges so you can try your trading strategies before using real funds.
```
Comment (gbeced) [inline README.md:13]: It should be *Backtesting exchange*, not exchanges.

**Comment by kamilkrukowski** [comment]
Updated per comment.

Happy to help. Project looks interesting and I plan on taking a more in-depth look.

**Review by gbeced — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (xzmeng) [reply README.md]: What does "Backtesting exchange" mean ...

Comment (gbeced) [reply README.md]: It means that, apart from Bitstamp and Binance, Basana also ships with a backtesting exchange class as documeted here: https://basana.readthedocs.io/en/latest/backtesting_exchange.html#basana.backtesting.exchange.Exchange

**Pull request merged**